addOns/client/CHANGELOG.md

```diff
@@ -8,7 +8,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [0.13.0] - 2025-02-04
 ### Added
-- Added support for Client Script Authentication when installed in conjunction with the Authentication Helper add-on. 
+- Added support for Client Script Authentication when installed in conjunction with the Authentication Helper add-on.
+- Add an API action to export the Client Map.
```

Comment:
This should be moved up to the Unreleased section as the add-on has been released while this has been being worked on.